hazelcast/src/test/java/com/hazelcast/internal/cluster/impl/MembershipUpdateTest.java

```diff
@@ -852,7 +852,7 @@ public void run() {
     @Test
     public void connectionsToRemovedMember_shouldBeClosed() {
         Config config = new Config()
-            .setProperty(GroupProperty.MAX_NO_HEARTBEAT_SECONDS.getName(), "10")
+            .setProperty(GroupProperty.MAX_NO_HEARTBEAT_SECONDS.getName(), "20")
```

Comment:
This will prolong the time the test takes in normal execution, because we rely on heartbeat timeout to detect missing member (see the `assertClusterSizeEventually` in line 873 below).
The test failure in #15710 failed in the first assert of cluster size (line 862 - just waiting for a cluster of 3 members to be formed during test setup). If there is evidence of long hiccups in the test failure logs, then I think we can close the issue without merging any change to the test itself.

Reply:
Yes, I am aware that this indeed prolongs the time to run the test (after all, this was the reason why the test set the timeout to 10s instead of using the default). I am fine with not increasing the timeout.